packages/mdc-checkbox/index.js

```diff
@@ -64,6 +64,7 @@ export class MDCCheckbox extends MDCComponent {
 
   getDefaultFoundation() {
     return new MDCCheckboxFoundation({
+      shouldIgnoreRippleActivation: () => this.disabled,
```

Comment:
Hmm....I'm thinking that `isSurfaceDisabled` or `isSurfaceInteractive` might be a better method name here. 

With `shouldIngoreRippleActivation`, that kind of leaves the heuristics of when a ripple should not be triggered up to the _implementor_. I'm unsure if this is what we should be going for. Instead, we should encode the heuristics of when the ripple should be activated into the foundation itself (e.g. when `isSurfaceActive` is false and we encounter a keyboard event, we don't enable the ripple).